sal_interfaces/MTMount/MTMount_Commands.xml

```diff
@@ -3,201 +3,127 @@
   <SALCommand>
     <Subsystem>MTMount</Subsystem>
-    <Version>3.0</Version>
-    <Author>Ander Ansuategi</Author>
     <EFDB_Topic>MTMount_command_trackTarget</EFDB_Topic>
-    <Alias>trackTarget</Alias>
-    <Device>target</Device>
-    <Property>position</Property>
-    <Action/>
-    <Value/>
-    <Explanation>http://sal.lsst.org/SAL/Commands/MTMount_command_trackTarget.html</Explanation>
+    <Description>Specify position, velocity and time (PVT) vector for all axes. This command is immediately reported as done.</Description>
     <item>
-      <EFDB_Name>az_angle</EFDB_Name>
-      <Description>Azimuth angle</Description>
+      <EFDB_Name>azimuth</EFDB_Name>
+      <Description>Mount azimuth angle at the specified time. The allowed range is the software limits (L1), approximately -270 to 270.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
+      <Units>deg</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>az_velocity</EFDB_Name>
-      <Description>Azimuth velocity</Description>
+      <EFDB_Name>azimuthVelocity</EFDB_Name>
+      <Description>Mount azimuth velocity at the specified time.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>degree/second</Units>
+      <Units>deg/second</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>el_angle</EFDB_Name>
-      <Description> Elevation angle</Description>
+      <EFDB_Name>elevation</EFDB_Name>
+      <Description>Mount elevation angle at the specified time.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
+      <Units>deg</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>el_velocity</EFDB_Name>
-      <Description> Elevation velocity</Description>
+      <EFDB_Name>elevationVelocity</EFDB_Name>
+      <Description>Mount elevation angle velocity at the specified time.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>degree/second</Units>
+      <Units>deg/second</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>time</EFDB_Name>
+      <EFDB_Name>taiTime</EFDB_Name>
       <Description>TAI date at which the commanded positions and velocities apply, as unix time.</Description>
       <IDL_Type>double</IDL_Type>
       <Units>second</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>cablewrap_orientation</EFDB_Name>
-      <Description>Cable Wrap orientation: Clockwise(CW), Counterclockwise(CCW)</Description>
+      <EFDB_Name>trackId</EFDB_Name>
+      <Description>Target identifier provided by the MTMount.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>dimensionless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>tracksys</EFDB_Name>
+      <Description>Defines tracking coordinate system as sidereal, non-sidereal or local.</Description>
       <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
+      <Units>dimensionless</Units>
```

Comment:
Why `dimensionless` instead of `unitless`? Same for line 91. `unitless` still is used in other places in this file.

Reply:
I'll standardize on unitless, though I prefer the former because it is Astropy's own standard.